src/vs/platform/contextview/browser/contextMenuHandler.ts

```diff
@@ -134,7 +137,13 @@ export class ContextMenuHandler {
 					this.block = null;
 				}
 
-				this.focusToReturn?.focus();
+				this.blockDisposable?.dispose();
```

Comment:
worth setting back to `null` so the dispose doesn't get run again in the render function?